rubygem-hammer_cli_katello.spec

```diff
@@ -52,7 +51,7 @@ gem install --local --install-dir .%{gem_dir} \
 
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d
-install -m 755 %{SOURCE1} %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/katello.yml
+install -m 755 .%{gem_instdir}/config/katello.yml %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/katello.yml
```

Comment:
i can't actually build the rpm with this change.  For one i think its geminstdir

but even that doesn't get you all the way